doc-content/enterprise-only/processes/using-keystore-vault-proc.adoc

```diff
@@ -0,0 +1,41 @@
+[id='using-keystore-vault-proc-{context}']
+= Using Keystore Vault
+
+You can use the Quarkiverse File Vault extension for storing credentials under the `file` method. For more information, see https://github.com/quarkiverse/quarkus-file-vault[Quarkiverse File Vault] and https://github.com/kiegroup/process-migration-service#using-keystore-vault[Using Keystore Vault] on GitHub.
+
+.Procedure
+
+. To add passwords to the Process Instance Migration (PIM) Keystore Vault, use the `keytool` command. For example:
++
+[source]
+----
+$ keytool -importpass -alias pimdb -keystore pimvault.p12 -storepass password -storetype PKCS12
+$ keytool -importpass -alias kieserver -keystore pimvault.p12 -storepass password -storetype PKCS12
+$ keytool -importpass -alias cert -keystore pimvault.p12 -storepass password -storetype PKCS12
+$ keytool -importpass -alias keystore -keystore pimvault.p12 -storepass password -storetype PKCS12
+$ keytool -importpass -alias truststore -keystore pimvault.p12 -storepass password -storetype PKCS12
+----
+. Configure the vault to use the keystore. For example:
++
+[source, yaml]
+----
+quarkus:
+  file:
+    vault:
+      provider:
+        pim:
+          path: pimvault.p12
+          secret: ${vault.storepassword} # This will be provided as a property
```

Comment:
What about adding an example of running pim with a provided property?
As an environment variable
```
VAULT_STOREPASSWORD=mysecret java -jar quarkus-app/quarkus-run.jar
```
As a system property
```
java -Dvault.storepassword=password -jar quarkus-app/quarkus-run.jar 
```